script.js

```diff
@@ -106,4 +106,9 @@ resetBtn.addEventListener("click", (event) => {
   billInput.value = "";
   peopleInput.value = "";
   customTipInput.value = "";
+  
+// remove the active state from tipsButton
+  tipButtons.forEach((node) => {
+    node.classList.remove("button-active")
+  })
```

Comment:
Good job. Two last things:

* I need you to reset the tip percentage to its default value (which is 5) so that variable doesn't retain the last value it was assigned, which could cause confusions. 

* I need you to select the 5% button on reset, and not remove the `button-active` from all the buttons.

Reply:
Don't forget to run pull changes before making yours so you don't get a merge conflict as I've added a commit since the last time you committed. 
You can ask any questions if you're confused about the anything.

Reply:
The default Tip-percentage was 15%. 
I will try to make 5% the default

Reply:
Ok, I will make pull changes